platform/linux/README.md

```diff
@@ -0,0 +1,17 @@
+HOWTO Use Virtual Switch
+
+
+1. Create a docker with 32 front panel port
+
+```
+docker run -id --name sw debian bash
+sudo ./create_vnet.sh sw
+```
+
+2. Create sonic virtual switch docker
+
+```
+docker run --privileged --network container:sw -d docker-sonic-linux
```

Comment:
>docker-sonic-linux [](start = 50, length = 18)

Call it docker-virtual-switch?

Reply:
changed to docker-sonic-vs

I need the sonic since this is the docker image that contains all sonic components